app/src/renderer/views/Inbox/MainContent/Conversation/Item/File.tsx

```diff
@@ -14,21 +14,176 @@ import "../Item.css";
 import "./File.css";
 
 import { useTranslation } from "react-i18next";
-import {
-  File as FileIcon,
-  Download,
-  LoaderCircle,
-  FileCheck2,
-  FileX2,
-} from "lucide-react";
+import { memo } from "react";
+import { File as FileIcon, Download, LoaderCircle, FileX2 } from "lucide-react";
 import { Button, Tooltip } from "antd";
+import {
+  FilePdfFilled,
+  FileExcelFilled,
+  FileImageFilled,
+  FileWordFilled,
+  FileZipFilled,
+  FilePptFilled,
+  FileMarkdownFilled,
+  FileFilled,
+} from "@ant-design/icons";
+import FileVideoFilled from "./FileVideoFilled";
+import FileAudioFilled from "./FileAudioFilled";
+
+const MemoizedFileVideoFilled = memo(FileVideoFilled);
```

Comment:
Any reason not to do this where the component is defined like we've done with the other ones?